services/collaboration/pkg/server/http/server.go

```diff
@@ -77,6 +78,11 @@ func Server(opts ...Option) (http.Service, error) {
 
 	prepareRoutes(mux, options)
 
+	_ = chi.Walk(mux, func(method string, route string, handler stdhttp.Handler, middlewares ...func(stdhttp.Handler) stdhttp.Handler) error {
+		options.Logger.Debug().Str("method", method).Str("route", route).Int("middlewares", len(middlewares)).Msg("serving endpoint")
+		return nil
+	})
```

Comment:
Do we want to keep this? Walking though the routes seems potentially expensive and I'm not sure if we want do that for something that should be fixed after development.
I mean, unless you're debugging during development, I don't think the routes will change, and I don't think we expect dynamic changes in production.

Reply:
ok, then maybe include a comment? Just to make sure we don't accidentally remove this